server/src/GernzyPaymentProvider.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Gernzy\Server;
+
+use Gernzy\Server\Packages\Paypal\PaypalProvider;
+use Gernzy\Server\Packages\Paypal\Services\PaypalService;
+use Gernzy\Server\Packages\Stripe\Services\StripeService;
+use Gernzy\Server\Packages\Stripe\StripeProvider;
+
+class GernzyPaymentProvider extends GernzyServiceProvider
+{
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        // Register dependency packages
```

Comment:
@AlexeiaW for enabled / registered payment providers could this rather feed in from a config value? Something like:

```
paymentProviders: [
   StripeProvider:: class,
   PaypalProvider::class
]
```

And then in the PaymentProvider we just loop through those from config. 

Ideally we keep the developer who is setting up a new Gernzy project from getting into code before they have to, so you only worry about composer dependencies and modifying config.

Reply:
Ok, made this config based

Reply:
one thing is I am struggling to access the service class associated with a payment provider.. how can we get to whatever service class is bonded in the provider boot method?